ci/pod/docker-compose.common.yml

```diff
@@ -102,3 +102,103 @@ services:
       VAULT_DEV_ROOT_TOKEN_ID: root
       VAULT_DEV_LISTEN_ADDRESS: 0.0.0.0:8200
     command: [ "vault", "server", "-dev" ]
+
+
+  ## RedisCluster Enable TLS
+  redis-node-0:
+    image: docker.io/bitnami/redis-cluster:7.0
+    volumes:
+      - ./t/certs:/certs
+    environment:
+      - 'ALLOW_EMPTY_PASSWORD=yes'
+      - 'REDIS_NODES=redis-node-0 redis-node-1 redis-node-2 redis-node-3 redis-node-4 redis-node-5'
```

Comment:
Could we reduce the number of redis nodes used in the CI?

Reply:
for a redis cluster, it may need at least 3 master nodes so i not reduce. But it might be my problem, can you give some advice, thx.

```
redis-node-1_1  | Waiting 0s before querying node ip addresses
redis-node-1_1  | *** ERROR: Invalid configuration for cluster creation.
redis-node-1_1  | *** Redis Cluster requires at least 3 master nodes.
```